subsys/fs/fs.c

```diff
@@ -160,6 +160,21 @@ int fs_open(struct fs_file_t *zfp, const char *file_name, fs_mode_t flags)
 		return -ENOTSUP;
 	}
 
+	bool truncate_file = false;
```

Comment:
Nit: move this to the top of the function.

Reply:
Done.